core-feature-pack/galleon-feature-pack/pom.xml

```diff
@@ -0,0 +1,372 @@
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.wildfly.core</groupId>
+        <artifactId>wildfly-core-feature-pack-parent</artifactId>
+        <version>13.0.0.Beta1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>wildfly-core-galleon-pack</artifactId>
+
+    <name>WildFly: Core Galleon Pack</name>
+    <packaging>pom</packaging>
+
+    <properties>
+        <license.directory>${project.build.directory}/resources/content/docs/licenses</license.directory>
+        <common.resources.directory>${basedir}/../common/src/main/resources</common.resources.directory>
+        <ee-8-api.resources.directory>${basedir}/../ee-8-api/src/main/resources</ee-8-api.resources.directory>
+        <legacy.resources.directory>${basedir}/../legacy-feature-pack/src/main/resources</legacy.resources.directory>
+        <galleon-common.resources.directory>${basedir}/../galleon-common/src/main/resources</galleon-common.resources.directory>
+        <common.license.directory>${basedir}/../common/src/license</common.license.directory>
+        <ee-8-api.license.directory>${basedir}/../ee-8-api/src/license</ee-8-api.license.directory>
+    </properties>
+
+    <dependencies>
+
+        <dependency>
+            <groupId>org.wildfly.galleon-plugins</groupId>
+            <artifactId>wildfly-galleon-plugins</artifactId>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.wildfly.galleon-plugins</groupId>
+            <artifactId>wildfly-config-gen</artifactId>
+            <scope>provided</scope>
+        </dependency>
+
+        <!-- module and copy artifact dependencies -->
+
+        <!-- Bring in all the common deps in one go -->
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>wildfly-core-feature-pack-common</artifactId>
+            <type>pom</type>
+            <scope>provided</scope>
+            <!-- Exclude the ee-8-api dependencies -->
```

Comment:
I'm not sure to follow.  `wildfly-core-feature-pack-common` "holds content that is expected to be usable in the legacy fp, the current EE 8 galleon FP, and a future EE 9 galleon FP". It depends on ee-8-api though but we exclude them in this feature pack to bring them back with wildfly-core-feature-pack-ee-8-api.

The comments seems to imply that this is for all ee-8-api dependencies, but you only exclude the JSON API.

Reply:
I only excluded json because that was the one that led to a ban-transitive-dependencies failure.  Excluding the others too would make sense just to avoid confusion or having to do it later.

It does make me wonder why there was no error reported for any of the others.

Reply:
Hmm, I thought you were asking about the exclusion in the 'common' module pom. TBH I forget exactly why this was needed; I'll investigate.

Reply:
I removed it and building with -rf core-feature-pack worked fine so it seems this wasn't necessary.

In the 'common' pom.xml I went ahead and excluded all the EE API artifacts.